### PR TITLE
Hebrew

### PR DESCRIPTION
**Pull request by davidpodless**
Starting to translate the Rules book to Hebrew

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,16 @@ Click in the table to download the most recent builds in the chosen language:
             <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/cs/printable_cs.pdf">stáhnout</a></td>
             <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/cs/economy_printable_cs.pdf">stáhnout</a></td>
         </tr>
+        <tr>
+            <td>🇮🇱 עברית</td>
+            <td align="center">10%</td>
+            <td align="center">0%</td>
+            <td align="center">—</td>
+            <td align="center">-</td>
```
Comment (qwrtln) [inline README.md:114]:
```suggestion
            <td align="center">—</td>
```

**Comment by qwrtln** [comment]
Please, post pictures of the text that was already translated. Also, please update `.github/CODEOWNERS` file.

**Comment by qwrtln** [comment]
Also, please target the PR to the main branch, so that the workflows can trigger, and we can see if the file builds properly.

**Comment by davidpodless** [comment]
> Please, post pictures of the text that was already translated. Also, please update `.github/CODEOWNERS` file.

image
image
image

There are still a lot of work TBD here. 
I also have few challenges I don't really know how to fix (Few graphic issue, and few numbers going RTL instead of LTR, you can see an example for that in the 3rd img)

**Comment by davidpodless** [comment]
> Also, please target the PR to the main branch, so that the workflows can trigger, and we can see if the file builds properly.

Shouldn't I wait for all the translation to be ready first?

File: main_he.tex [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+% Language setting
+\usepackage{fontspec} % For font settings
+\usepackage{polyglossia} % Better for handling multiple languages with XeLaTeX or LuaLaTeX
+
+% Set languages
+\setmainlanguage{hebrew} % Set the main language to Hebrew
+\setotherlanguage{english} % Optional: if you have English text as well
+
+% Set main font for Latin script (English)
+\setmainfont{Times New Roman}
```
Comment (qwrtln) [inline main_he.tex:14]: Why? Original LaTeX font doesn't work?

File: main_he.tex [hunk 3]
```diff
@@ -0,0 +1,61 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+
```
Comment (qwrtln) [inline main_he.tex:4]:
```suggestion
```

File: main_he.tex [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+% Language setting
+\usepackage{fontspec} % For font settings
+\usepackage{polyglossia} % Better for handling multiple languages with XeLaTeX or LuaLaTeX
```
Comment (qwrtln) [inline main_he.tex:7]: Planning to get rid of it? Or is it required? I guess there will always be a tiny Latin component, like author, credits, right?

File: main_he.tex [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  Script=Hebrew,
+  Scale=MatchLowercase
+]
+
+% If you need to use monospace font for both scripts
+\setmonofont{DejaVu Sans Mono}
+
+
+
+
```
Comment (qwrtln) [inline main_he.tex:33]:
```suggestion
```
Do you really need this? We're not writing any code or math formulas here. Also, please get rid of excessive blank lines.

File: main_he.tex [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+\def\sections{sections/translated/he}
+\def\lang_header_adjustment{0px}
+
+\newcommand{\version}{
+  גרסה \textenglish{1.2}
```
Comment (qwrtln) [inline main_he.tex:38]: Are you sure about this? There's a different version handling now.

Comment (davidpodless) [reply main_he.tex:38]: I just copied what I saw when I wrote this, when there will be a new version I'll update it too

File: main_he.tex [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+\newcommand{\intro}{
+זהו פרוייקט קהילתי מבוסס \href{https://github.com/Heegu-sama/Homm3BG}{גיט}.
+ כלל הציבור מוזמים להשתתף, לשנות ולתקן שגיאות.
+אם ברצונך להעיר הערות, ניתן לעשות זאת \href{https://boardgamegeek.com/thread/3235221/rule-book-rewrite-project}{בדיון ב-\textenglish{BoardGameGeek}}
+}
+\newcommand{\pageshorthand}{с.}
```
Comment (qwrtln) [inline main_he.tex:46]: This looks like Russian leftover (с. - сторона, in English it's p. -age). How should be in Hebrew?

File: main_he.tex (hunk 4 above)
Comment (davidpodless) [reply main_he.tex:7]: Few of the names, also - numbers.

File: main_he.tex (hunk 2 above)
Comment (davidpodless) [reply main_he.tex:13]: It didn't compiled for me without it (also it didn't work without \setmonofont{DejaVu Sans Mono})

I don't mind reworking on this file later on. for now I just wanted it to compile so I'll see the pdf and see what is done and what to fix in the translation itself

**Comment by qwrtln** [comment]
> > Also, please target the PR to the main branch, so that the workflows can trigger, and we can see if the file builds properly.
> 
> Shouldn't I wait for all the translation to be ready first?

No, we have a different workflow. Our approach is to have one PR per section. I'm encouraging you to do it like that. First PR should be just the setup, title page, and that's it. Every other section should get its own PR.

File: main_he.tex (hunk 7 above)
Comment (davidpodless) [reply main_he.tex:45]: ע.
Yup you are right, I searched how to change it and didn't find it

**Comment by davidpodless** [comment]
> > > Also, please target the PR to the main branch, so that the workflows can trigger, and we can see if the file builds properly.
> > 
> > 
> > Shouldn't I wait for all the translation to be ready first?
> 
> No, we have a different workflow. Our approach is to have one PR per section. I'm encouraging you to do it like that. First PR should be just the setup, title page, and that's it. Every other section should get its own PR.

So... should I break this PR to 7 PRs?
OK
so Let's not waste anymore time on this one :-)

**Comment by qwrtln** [comment]
> So... should I break this PR to 7 PRs? OK so Let's not waste anymore time on this one :-)

Yes, exactly. The first one should be just a scaffold for the translation. Believe me, it will make everyone's life easier. It's better to split the work into bite-size chunks, instead of a huge PR weeks behind the main branch.

**Comment by DarthGandalf** [comment]
Actually it's "страница", not what you said. And it's supposed to be стр. instead of с.

But that makes text of printable version much longer, and breaks the layout on many pages. With incorrect but short "c." the effect is not as severe

**Pull request closed without merging**